From 0fa24a67e87b5754bba02a503923eee7e4acbdf0 Mon Sep 17 00:00:00 2001
From: Madhavan Ganesan <quic_madhgane@quicinc.com>
Date: Wed, 13 Nov 2024 14:04:49 +0530
Subject: [PATCH] iscripts: Makefile: add __CONFIG_IO_COHERENCY__ to dtc flags

Add __CONFIG_IO_COHERENCY__ DT flag to build the necessary DTS property.

Change-Id: Ic384a635cbb4633ad04d79cfbf8c97c10f2dc3be
Signed-off-by: Madhavan Ganesan <quic_madhgane@quicinc.com>
---
 scripts/Makefile.lib | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index 610e3d1899cb..103607a5f699 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -273,6 +273,10 @@ ifeq ($(CONFIG_MHI_BUS_RESERVED_DMA_POOL),y)
 dtc_cpp_flags += -D __MHI_BUS_RESERVED_DMA_POOL__
 endif
 
+ifeq ($(CONFIG_IO_COHERENCY),y)
+dtc_cpp_flags += -D __CONFIG_IO_COHERENCY__
+endif
+
 ifdef CONFIG_OBJTOOL
 
 objtool := $(objtree)/tools/objtool/objtool
-- 
2.34.1

